src/pages/settings/Profile/ProfilePage.js

```diff
@@ -159,13 +160,14 @@ class ProfilePage extends Component {
     }
 
     validateInputs() {
-        const {firstNameError, lastNameError} = PersonalDetails.getFirstAndLastNameErrors(this.state.firstName, this.state.lastName);
-
+        const errors = PersonalDetails.getCharacterLimitErrors(50, [this.state.firstName, this.state.lastName, this.state.pronouns]);
+        const [firstNameError, lastNameError, pronounError] = errors;
```

Comment:
Hmm...

Very strange way of destructing ultimately killing the purpose of it. Please move this in place of `const errors` and remove the errors.